src/parser/core/Events.ts

```diff
@@ -330,6 +331,8 @@ export interface Event<T extends string> {
   prepull?: boolean;
   /** Other events associated with this event. Added by WoWA normalizers. Meaning is context sensitive */
   _linkedEvents?: LinkedEvent[];
+  // true if links have been processed for this event to prevent duplicate linking
```

Comment:
[very tiny grammar nit]
```suggestion
  // contains the links that have been processed for this event to prevent duplicate linking
```

Reply:
oh you already fixed it, damn

Reply:
Yea didn't update comment when i first linked but it should be good now